.github/build.in.yml

```
@@ -94,6 +94,26 @@ jobs:
               : # Do nothing on failure, but suppress errors
           fi
 
+      - name: cleanup .cache/mathlib
+        run: |
+          # Define the cache directory path
+          CACHE_DIR="$HOME/.cache/mathlib"
+
+          # Check if directory exists
+          if [ ! -d "$CACHE_DIR" ]; then
+            echo "::warning::Cache directory does not exist: $CACHE_DIR"
+            exit 0
+          fi
+
+          # Calculate directory size in bytes
+          DIR_SIZE=$(du -sb "$CACHE_DIR" | cut -f1)
```

```suggestion
          DIR_SIZE=$(du -sb "$CACHE_DIR" | cut -f1)
          printf '%s\n' "$DIR_SIZE"
```
to have some data available anyway.

What do you think?

---

Sounds good! (Claude originally had a snippet in its original suggestion to convert back to GB, but apparently `bc` isn't available on the runners, so I just went with your suggestion in the end instead.)

---

If you want human readable, maybe `du -h -d0` (untested) would work?